docs/source/installation/for_dev.rst

```diff
@@ -20,39 +20,12 @@ steps to clone your fork.
 For Developers with Poetry (Recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Poetry can be easily installed in any OS by running the following command:
+Why we recommend using poetry ?
 
-Linux / OSX / BashOnWindows install instructions
-************************************************
+Because Poetry is a tool for dependency management and packaging in Python. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you.
```

Comment:
```suggestion
Poetry is a tool for dependency management and packaging in Python. It allows to
declare the libraries a project depends on and it will manage (install/update) them
for you. In addition, Poetry provides a simple interface for managing virtual environments.
```